packages/util-stream-node/src/sdk-stream-mixin.ts

```diff
@@ -32,7 +32,7 @@ export const sdkStreamMixin = (stream: unknown): SdkStream<Readable> => {
     transformToString: async (encoding?: string) => {
       const buf = await transformToByteArray();
       if (encoding === undefined || Buffer.isEncoding(encoding)) {
-        return fromArrayBuffer(buf.buffer, buf.byteOffset, buf.byteLength).toString(encoding);
+        return fromArrayBuffer(buf.buffer, buf.byteOffset, buf.byteLength).toString(encoding as BufferEncoding);
```

Comment:
I am telling Buffer.toString to not typecheck the encoding. This should not be applied upward to `encoding?: string` because
- I don't think the enumeration is actually exhaustive, but regardless,
- it makes it very hard to make the browser version consistently typed with this one